src/pages/searchPage/CloseBtnAnimation.tsx

```diff
@@ -0,0 +1,30 @@
+import styled, { css, keyframes } from "styled-components";
+
+const fadeInWithGoRight = keyframes`
+    0% {
+        opacity: 0;
+        transform: translateX(-40px) rotate(-30deg);
+    }
+    100% {
+        opacity: 1;
+        transform: translateX(0px) rotate(0deg);
+    }
+`;
+
+const animationBezier = css`
+  animation-timing-function: cubic-bezier(0, 0.51, 0.38, 1);
+  animation-duration: 0.5s;
+`;
+
+const AnimationWrapper = styled.div`
+  animation-name: ${fadeInWithGoRight};
+  ${animationBezier}
+`;
```

Comment:
재사용 안하면 그냥 한번에 작성해도 좋을 것 같아요
```suggestion
const AnimationWrapper = styled.div`
  animation: ${fadeInWithGoRight} 0.5 cubic-bezier(0, 0.51, 0.38, 1);
`;
```

Reply:
음 확실히 그럴것같네요